frontend/specs/learner/launch-and-complete-study.spec.ts

```diff
@@ -23,7 +23,7 @@ test('launching study and testing completion', async ({ browser }) => {
     await userPage.getByText('All Studies').isVisible()
     await userPage.getByText(studyName).first().click()
 
-    await userPage.click('testId=launch-study')
+    // await userPage.click('testId=launch-study')
```

Comment:
any specific reason why this is removed now?

Reply:
Devika mentioned that the study should be launched when the study card is clicked. `await userPage.click('testId=launch-study')` clicks the button on the study details preview, which no longer exists, hence I removed it

Reply:
if so - then we can just remove this line rather than comment it out